woocommerce/woocommerce-gutenberg-products-block
# Product Gallery: Lock the Sale Badge and the Next/Prev Buttons

## What

Fixes #10868

- The lock on the `Sale Badge` still allows the user to delete the block.
- For the `Next/Prev Buttons` such thing is not necessary as the user can hide the block with block settings.

## Why

This PR locks the `Sale Badge` and `Next/Prev Buttons` blocks in order to prevent unexpected behavior when these blocks get moved.


## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Using a block theme, go to `Appearance > Editor`.
2. Go to `Templates > Single Product`.
3. Make sure you are using the blockified template.
4. Add the **Product Gallery** block .
5. Try to move the `Sale Badge` below the `Next/Prev Buttons`.
6. Notice the blocks are locked.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|<img width="825" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress-3" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/298e66d9-ff87-4272-8434-a339c152abee">|<img width="788" alt="Single_Product_‹_Template_‹_Editor_‹_newproductgallery_—_WordPress-2" src="https://github.com/woocommerce/woocommerce-blocks/assets/905781/3d17c6a7-f87a-4a59-9432-8cc45667a347">|

## WooCommerce Visibility


Required:


## Checklist

Required:

Conditional:

## Changelog


> Product Gallery: Lock the Sale Badge and Next/Prev Buttons.
